app/controllers/authentication_controller.rb

```diff
@@ -7,7 +7,7 @@ def authenticate
     if command.success?
       render json: {
         auth_token: command.result,
-        user: command.authenticated_user
+        user: command.authenticated_user.slice(:id, :first_name, :last_name, :email)
```

Comment:
can't use UserSerializer here because it will just generate the body of an entire User data response. But this portion is already within a JSON response.